contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -70,6 +92,7 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
      * @return 0 if the redeem is allowed, otherwise a semi-opaque error code (See ErrorReporter.sol)
      */
     function redeemAllowed(address vToken, address redeemer, uint256 redeemTokens) external returns (uint256) {
+        ensureUserAllowed(redeemer);
```

Comment:
Info: in the Isolated pools (not this repo), we should also check if the caller is allowed to redeem vTokens on behalf of the redeemer